README.md

```diff
@@ -50,10 +50,18 @@ Then, we will work on BPMN extensions, library extension points, display options
 capacities.
 
 
-# Development
+# Contributing
```

Comment:
what is the purpose of this change? do you prefer having minimal info here because you think it makes easier to start contributing?
In that case, call to `npm install` is missing I guess

Reply:
I got inspired by different readme, like https://rowanmanning.com/posts/writing-a-friendly-readme/
This provides a taste for new contributors, and encourages them to read more.